COMPOSIO_MIGRATION.md

```
@@ -0,0 +1,253 @@
+      "description": "Composio configuration for Mapbox integration"
+    }
+  }
+}
+```
+
+### 4. Code Changes
+
+#### mapbox_mcp/hooks.ts
+
+**Old Connection Method:**
+```typescript
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)